src/components/space/spaceBox/SpaceBox.tsx

```diff
@@ -25,11 +28,27 @@ export const SpaceBox = () => {
     setCurrentSpace(userSpaces[account]?.spaces[selectedSpace]);
   }, [selectedSpace]);
 
+
+  useEffect(()=>{
+    getCurrentTime();
+  },[]);
+
+  const getCurrentTime = async () => {
+    var today = new Date();
+    //date in the between params will be scheduleAt and scheduleEnd for the 
+    let howWork = moment(today).isBetween('2023-01-01', '2023-10-25'); 
+    await setActive(howWork)
+    // return howWork;
+  }
```

Comment:
There is a method in helper/space/spaceUtility folder for checking space time. that can be used here.